Jenkinsfile

```diff
@@ -1,5 +1,23 @@
 #!groovy
 
+def genBranch(String arch) {
+	return [
+		"${arch}": { ->
+			wrappedNode(label: "linux&&${arch}", cleanWorkspace: true) {
+				safeGithubNotify(status: 'PENDING', context: "${arch}")
```

Comment:
You'll need to add this function if you want this to work

Reply:
Thanks for looking out!